index.js

```diff
@@ -0,0 +1,87 @@
+const players = "http://localhost:3000/players"
```

Comment:
I might use a different variable name here, like `playerUrl`, to make it clear what this variable is being used for.